ang/prospect/app.config.js

```diff
@@ -0,0 +1,43 @@
+(function (angular, $, _) {
+  var module = angular.module('prospect');
+
+  module.run(function () {
+    (function init () {
+      CRM.civicase.caseActions = CRM.civicase.caseActions.concat(getCaseActionsForProspect());
```

Comment:
This will add the actions to the end of the list, is that ok?

Reply:
There was not any clear instruction on where to end the new actions. But I changed it to appear before others 👍

Reply:
I was just asking :3